src/components/list/ListPage.tsx

```
@@ -112,16 +67,15 @@ export default function ListPage({ type, categoryId, searchWord }: ListPageProps
     loadProjects();
-    setTotalCount(999999);
-    // 페이지 불러오기 로직
-    // currentPage 변경, morePage 변경
   }, []);
 
   useEffect(() => {
-    setCurrentPage(0);
-    loadProjects();
-    setTotalCount(999999);
-    // 새 정렬로 페이지 불러오기 로직
+    const resetAndLoad = async () => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: loadProjects

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)